.env.yjp.example

```diff
@@ -0,0 +1,17 @@
+# This is a sample docker compose env file to run the geoserver services
+# with the Yourkit Java Profiles connection broker.
+# Copy as .yjp.env and replace <broker-id> and <broker-token> with the values
+# sent to you by email.
+# Then run with docker compose --env-file .env --env-file .env.yjp -f ... up -d
+# See https://www.yourkit.com/docs/connection-broker/2023.9/help/
+
+YJP_OPTS=-agentpath:/usr/local/YourKit-JavaProfiler-2023.9/bin/linux-x86-64/libyjpagent.so=broker_url=https://broker.yourkit.com/<broker-id>/,broker_token=<broker-token>
```

Comment:
may need to revisit in the context of multiplatform docker images (https://github.com/geoserver/geoserver-cloud/pull/406)